model/messages/parsedmessage.go

```diff
@@ -22,7 +22,7 @@ type ParsedMessage struct {
 	Value  string `pg:",notnull"`
 	Method string `pg:",notnull"`
 
-	Params []byte `pg:",type:jsonb,notnull"`
+	Params string `pg:",type:jsonb,notnull"`
```

Comment:
This is the fix for the below errors, yeah?

```
2020-10-19T21:24:36.842Z        ERROR   message message/message.go:107  failed to process tipset        {"error": "process tipset: persist: persisting parsed messages: ERROR #22P02 invalid input syntax for type json", "height": 161014}
```

Reply:
yes